shell/packages/sandstorm-db/migrations.js

```diff
@@ -305,5 +341,4 @@ function migrateToLatest() {
   }
 }
 
-// Apply all migrations on startup.
-Meteor.startup(migrateToLatest);
+SandstormDb.prototype.migrateToLatest = migrateToLatest;
```

Comment:
Don't worry about this right now, but this appears to work because `SandstormDb` is defined in a file alphabetically before "migrations.js" - we should probably make the ordering more explicit.

I could see a SandstormDb getting declared in `lib/db.js`, and methods on that object or on that prototype being defined in `common/` or `client/` or `server/` depending on where they're safe to use, since we have a number of methods that are client-only or server-only.